### ISSUE TITLE
window.webkit is undefined

### ISSUE THREAD

**Issue by kmurf1999** [Banno/banno-plugin-framework-bridge#6]
This is likely a problem with the parent IOS application and not this library.

After opening a link using `window.webkit.messageHandlers` in IOS and then subsequently returning to the Banno mobile application, `window.webkit` becomes undefined.  Because `window.webkit` is undefined, we are then unable to open any new links

**Comment by kmurf1999** [comment]
My team is currently developing an application that relies heavily on this functionality.
- Is this a known issue?
- Is there an available workaround?

**Comment by kmurf1999** [comment]
Sometimes, refreshing the IOS application by pulling down, makes `window.webkit` reappear.  though it's pretty inconsistent.

**Comment by jrobinson01** [comment]
@kmurf1999 Are you opening the links from within a plugin card? If so can you give me an example link? I'm getting setup to reproduce this on our end.

**Comment by kmurf1999** [comment]
Yes, I am.  We're using this library within our React.js app.  We're getting the same error on all links, but here is an example:

```js
<a
    href={completeUrl}
    style={{ background: color }}
    className={styles.redirectButton}
    target='_blank'
    onClick={() => setHasRedirected(true)}
>
  {initialButtonText}
</a>
```

We are also hosting multiple instances of our widget on the same page if that makes a difference (one for development and production)

**Comment by kmurf1999** [comment]
We've also tried calling the `postUniversalMessage` function manually and are having the same issue.

**Comment by kmurf1999** [comment]
@jrobinson01 The seems to have been fixed.  Can you confirm that a fix has been deployed for this and I can close this issue?

**Comment by jrobinson01** [comment]
@kmurf1999 did you update the ios app by any chance?

**Comment by kmurf1999** [comment]
Yes, I've updated to version 2.37.0 which was released 5 days ago

**Comment by jrobinson01** [comment]
Ok cool, from what I've gathered, this should have been fixed as of 2.34.0. I'm going to close the issue since you're no longer experiencing the problem.

**Comment by kmurf1999** [comment]
We're seeing this issue resurface intermittently in 2.37.0
